bot.py

```diff
@@ -50,6 +52,21 @@
     else:
         await client.send_message(message.channel, 'The current song is : "{}"'.format(elevator.title))
 
+async def playSound(message, soundURL):
+    if not in_elevator:
+        voice_channel = client.get_channel(message.author.voice.voice_channel.id)
+        voice_client = await client.join_voice_channel(voice_channel)
+        genji = await voice_client.create_ytdl_player(soundURL)
+        genji.volume = .2
+        genji.start() 
+        genji.sleep(5) #idk if theres a "wait for playback finish"
+
+    if not in_elevator:
+        await client.voice_client_in(message.server).disconnect()
+    else:
+    	elevatorChannel = discord.utils.get(after.server.channels, name=elevatorName)
+        await client.move_member(message.server, elevatorChannel)
```

Comment:
You should restart the elevator music after you move the client back into the elevator channel.